buildkite/client/pipeline.go

```diff
@@ -246,12 +244,12 @@ query Pipeline($slug: ID!) {
 		return nil, err
 	}
 
-	teamUUIDs := make([]string, len(resp.Pipeline.Teams.Edges))
-	for i, UUID := range resp.Pipeline.Teams.Edges {
-		teamUUIDs[i] = UUID.Node.Team.UUID
+	teamIDs := make([]string, len(resp.Pipeline.Teams.Edges))
+	for i, ID := range resp.Pipeline.Teams.Edges {
```

Comment:
Up yo you but perhaps `edge` is a better name for this in-loop variable?

```suggestion
	for i, edge := range resp.Pipeline.Teams.Edges {
```

Otherwise, line 249 reads a bit weird due to the redundant `ID` bits:
```
teamIDs[i] = ID.Node.Team.ID
```